src/headless/tasks/inasafe_wrapper.py

```diff
@@ -281,3 +286,46 @@ def check_broker_connection():
     :return: True
     """
     return True
+
+
+@app.task(
+    name='inasafe.headless.tasks.push_to_geonode',
+    queue='inasafe-headless-geonode')
+def push_to_geonode(
+        layer_uri,
+        geonode_url=REALTIME_GEONODE_URL,
+        geonode_user=REALTIME_GEONODE_USER,
+        geonode_password=REALTIME_GEONODE_PASSWORD):
+    """Upload layer to geonode instance.
+
+    :param layer_uri: The uri to the layer.
+    :type layer_uri: basestring
+
+    :param geonode_url: The URL to the GeoNode instance (including protocol).
```

Comment:
you can give example in this docstring

Reply:
Example of what? I have removed the parameters except for the layer_uri.